x-pack/plugins/security_solution/common/api/asset_criticality/get_asset_criticality.schema.yaml

```diff
@@ -0,0 +1,22 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Asset Criticality Get Record Schema
+paths:
+  /internal/asset_criticality:
+    get:
+      summary: Get Criticality Record
+      parameters:
+        - $ref: '#/components/parameters/id_value'
+        - $ref: '#/components/parameters/id_field'
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SingleAssetCriticality'
```

Comment:
Requesting a single criticality record is useful for the UI, but for scoring we need an efficient way to retrieve multiple records. I've started to build a WIP function to do this, but I'm not sure where it's ultimately going to live yet. You can check it out on [my branch](https://github.com/hop-dev/kibana/compare/8085-asset-criticality-apis...rylnd:kibana:asset_scoring_rebased?expand=1) (The above link is from a rebased version that includes this branch, so the diff might start to look weird as you add more to this branch).

Let's sync up on this and make sure we're going to meet in the middle.

Reply:
> a multi get route would live at POST /internal/risk_score/criticality/search if we came to implement one?

Yeah, the only real interesting question was about how we represent the request for multiple criticality records. If we built a general endpoint we could just allow a subset of ES DSL, and at that point it doesn't look much different from a search strategy 🤷 .